### ISSUE TITLE
Issue FDS 6.9.1 - Program Exception - access violation

### ISSUE THREAD

**Issue by Perry-FSC** [firemodels/fds#13111]
Good morning,
I'm new to this forum, so I apologize in advance if I haven't mastered all the features.
I'm having issues with a simulation I'm currently working on.
A fire model with 701k cell  with a mesh divided into 64 parts, corresponding to the 64 physical cores of my computer. 
cell size, initially, 0.5x0.5x0.5m.

I want to carry out a 900s modeling, but systematically at 500s or 600s, FDS crashes with the following error message:

"....
Time Step: 6700, Simulation Time: 500.92 s
 Step Time: 6800, Simulation Time: 506.08 s
forrtl: severe (157): Program Exception - access violation
PC Image Routine Line Source
fds_openmp.exe 00007FF688D65B11 Unknown Unknown Unknown
fds_openmp.exe 00007FF688A67BC9 Unknown Unknown Unknown
fds_openmp.exe 00007FF688A25DCB Unknown Unknown Unknown
fds_openmp.exe 00007FF6893DEBB0 Unknown Unknown Unknown
KERNEL32.DLL 00007FFA17FD257D Unknown Unknown Unknown
ntdll.dll 00007FFA1914AF28 Unknown Unknown Unknown"

I attach the fds file as an appendix. Hope is is accessible.
Can anyone tell me about the error message?
Many thanks in advance.

**Desktop :**
 - OS: Win11pro
 - AMD Ryzen Threadripper PRO 3995WX 64-Cores - 2.70 GHz
 - 64Gb RAM
 - FDS 6.9.1 with Pyrosim 2024.1.0611

[SC0_7+polycarbonates.txt](https://github.com/user-attachments/files/16066504/SC0_7%2Bpolycarbonates.txt)

**Comment by mcgratta** [comment]
I will run the case.

**Comment by mcgratta** [comment]
The case failed for me too. I'll try to see what is wrong.

**Comment by Perry-FSC** [comment]
Many thanks.
I retry with less mesh (32). With this configuration the fire is fully enclosed in 1 mesh.
I now reach 625s, another 1 hour of simulation and we will see

**Comment by Perry-FSC** [comment]
It's done.
I have now reached 900s of simulation in 1h43.
It works much better now with 32 meshes instead of 64.
It seems that the 126m² fire vent spread over two meshes is causing it to malfunction.

**Comment by mcgratta** [comment]
Actually, there are several possible reasons for the crash. `BURN_AWAY` is always a possibility, but in your case, when I removed it, the code still crashed. Then I removed `THICKEN=T` and that prevented the crash and the case ran to completion. I am not a fan of `THICKEN`. It's an old feature and really not necessary anymore except in very special cases. In your case, I suspect that various obstructions were thickening into each other, causing confusion when the obstructions were removed or burned away.

I'll use this as a test case to see if I can pinpoint the exact cause.
